src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -311,6 +319,33 @@ def load_migration_products_offline
         :next
       end
 
+      # check if the target producte has been already activated, if yes
+      # require user confirmation for the migration (though it will very likely fail)
+      # @param base_product [Y2Packager::Product] the base product to which migrate to
+      # @param activations [Array<SUSE::Connect::Remote::Product>] already activated
+      #   products on the server
+      # @return [Boolean] true = continue with the migration, false = stop
+      def migration_confirmed?(base_product, activations)
+        # version without the release, e.g. "15-0" => "15"
+        base_product_version = base_product.version.split("-", 2).first
```

Comment:
this is a bit confusing to use splitinto two  and then use just first one. Maybe use something like
```
base_product_version = base_product.version[/^[^-]*/]
# or less cryptic
base_product_version, _minor_version = base_product.version.split("-", 2)
```